vanilla_first_setup/views/user.py

```diff
@@ -128,3 +133,46 @@ def __verify_username(self) -> str:
             return _("This username is already in use.")
         
         return ""
+
+    def __verify_password(self) -> str:
+        password = self.password_entry.get_text()
+        confirmation = self.password_confirmation.get_text()
+
+        if not password:
+            return _("Password cannot be empty.")
+
+        if password != confirmation:
+            return _("Passwords do not match.")
+
+        forbidden_chars = {'&', ';', '|', '$', '>', '<', '!', '\\', '\t', '\n'}
```

Comment:
I'm not aware of any forbidden characters for linux passwords.

"\n" would be problematic though that's impossible to type in a password field so checking for that isn't really necessary.